data/database.py

```diff
@@ -74,6 +74,7 @@
     "sqlite": schemedriver(SqliteDatabase, PooledSqliteDatabase),
     "postgresql": schemedriver(PostgresqlDatabase, PooledPostgresqlDatabase),
     "postgresql+psycopg2": schemedriver(PostgresqlDatabase, PooledPostgresqlDatabase),
+    "cockroachdb": schemedriver(PostgresqlDatabase, PooledPostgresqlDatabase),
```

Comment:
Are we doing anything special for cockroachDB? Can we treat it as a generic postgres?

Reply:
unfortuantly we cannot during the migration especially as sqlalchemy gets mad.